README.md

```diff
@@ -133,6 +133,12 @@ This means that `./gradlew clean` and `./gradlew cleanExamples` will fail if `as
 Note that IntelliJ [does not support multiple projects in the same window](https://youtrack.jetbrains.com/issue/IDEABKL-6118#)
 so each sub-project must be opened in its own window.
 
+## Examples
+
+The `./examples` folder contain a number of example projects showing how Realm can be used. If this is the first time you checkout this repository to try the examples, you must call `./gradlew assemble` from the top-level directory first. Otherwise the examples will not compile as they depend on all Realm artifacts being installed in `mavenLocal()`.
```

Comment:
`installRealm` doesn't install the gradle plugin, so I don't think that is enough? `installRealmJava` installs both library and plugin, will need to check if that works.